eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/service/DbsnpClusteredVariantMonotonicAccessioningService.java

```diff
@@ -16,21 +16,30 @@
 package uk.ac.ebi.eva.accession.core.service;
 
 import uk.ac.ebi.ampt2d.commons.accession.core.BasicAccessioningService;
+import uk.ac.ebi.ampt2d.commons.accession.core.models.AccessionWrapper;
 import uk.ac.ebi.ampt2d.commons.accession.generators.monotonic.MonotonicAccessionGenerator;
 
 import uk.ac.ebi.eva.accession.core.IClusteredVariant;
 import uk.ac.ebi.eva.accession.core.persistence.DbsnpClusteredVariantAccessioningDatabaseService;
 
+import java.util.List;
 import java.util.function.Function;
 
 public class DbsnpClusteredVariantMonotonicAccessioningService
         extends BasicAccessioningService<IClusteredVariant, String, Long> {
 
+    private final DbsnpClusteredVariantAccessioningDatabaseService dbService;
+
     public DbsnpClusteredVariantMonotonicAccessioningService(
             MonotonicAccessionGenerator<IClusteredVariant> accessionGenerator,
             DbsnpClusteredVariantAccessioningDatabaseService dbService,
             Function<IClusteredVariant, String> summaryFunction,
             Function<String, String> hashingFunction) {
         super(accessionGenerator, dbService, summaryFunction, hashingFunction);
+        this.dbService = dbService;
+    }
+
+    public List<AccessionWrapper<IClusteredVariant, String, Long>> getByHash(List<String> hashes) {
```

Comment:
is this really needed? why don't you search using `AccessioningService::get(List<MODEL>)`

Reply:
it was not needed as get calculates the hashes and use them to query. Removed `getByHash` and used `get` instead